src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeMemcachedSessionHandler.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Session\Storage\Handler;
+
+/**
+ * Native session handler using php-memcached, PHP's Memcache extension (ext-memcached).
+ *
+ * @author Maurits van der Schee <maurits@vdschee.nl>
+ */
+class NativeMemcachedSessionHandler extends \SessionHandler
```

Comment:
Should be final

Reply:
I believe it shouldn't (following the style of the existing code), see also: https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeFileSessionHandler.php